modules/map/admin.html

```diff
@@ -0,0 +1,49 @@
+<form data-name="Text">
+	<label>Title<br>
+		<input type="text" data-target="title" placeholder="Title" value="Section">
+	</label>
+
+	<label>Google Map URL<br>
+		<input type="text" id="userEnteredMapUrl" data-target="userEnteredMapUrlTarget"  style="width: 300px;" placeholder="Search on Google Maps and Copy URL here">
+	</label>
+	
+	<input type="hidden" id="embedUrl" data-target="embedUrlTarget" value="">
```

Comment:
let's keep data-target values simple and consistent with other blocks, just call it `map`?